File: src/app/model/usuario.ts
import { Validators } from "@angular/forms";

export interface Usuario {
    idUsuario: number;
    login: string;
    contrasena: string;
    primerNombre: string;
    segundoNombre: string;
    primerApellido: string;
    segundoApellido: string;
    fechaNacimiento: Date;
    idEps: number;
    eps: string;
    celular: string;
    direccion: string;
    email: string;
    nombreContacto: string;
    telefonoContacto: string;
    parentescoContacto: string;
    fechaCreacion: Date;
}

export const USUARIOFORM = {
    login: ['', Validators.required],
    contrasena: ['', Validators.required],
    contrasenaRep: ['', Validators.required],
    primerNombre: ['', Validators.required],
    segundoNombre: ['', Validators.required],
    primerApellido: ['', Validators.required],
    segundoApellido: ['', Validators.required],
    fechaNacimiento: ['', Validators.required],
    idEps: [''],
    eps: ['', Validators.required],
    celular: ['', Validators.required],
    direccion: ['', Validators.required],
    email: ['', Validators.required],
    nombreContacto: ['', Validators.required],
    telefonoContacto: ['', Validators.required],
    parentescoContacto: ['', Validators.required]
}

export const USUARIOSESIONFORM = {
    login: ['', Validators.required],
    contrasena: ['', Validators.required]
}

File: back/vidaplena/vidaplena/src/main/java/com/vidaplena/vidaplena/aplicacion/login/dto/Usuario.java
package com.vidaplena.vidaplena.aplicacion.login.dto;

import org.springframework.stereotype.Repository;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.sql.Date;

@Getter
@Setter
@NoArgsConstructor
@ToString
@Repository
public class Usuario {
    private Long idUsuario;
    private String login;
    private String contrasena;
    private String primerNombre;
    private String segundoNombre;
    private String primerApellido;
    private String segundoApellido;
    private Date fechaNacimiento;
    private Long idEps;
    private String eps;
    private String celular;
    private String direccion;
    private String email;
    private String nombreContacto;
    private String telefonoContacto;
    private String parentescoContacto;
    private Date fechaCreacion;
}
